core/gui/app/services/mock-data.ts

```diff
@@ -222,6 +222,33 @@ let nltkSentiment = {
   }
 }
 
+let MLSentiment = {
```

Comment:
To make this operator available in the old GUI, you also need add this operator to the list in the very bottom of this file. Use the same ID in that list and in the HTML file (I saw you use `30`). You can try it first and see if it works.

If we don't care about the old GUI anymore, please remove these changes.